PWGGA/GammaConv/AliAnalysisTaskOmegaToPiZeroGamma.cxx

```diff
@@ -2447,6 +2503,24 @@ void AliAnalysisTaskOmegaToPiZeroGamma::CalculateOmegaCandidates()
                       if(fInputEvent->IsA()==AliAODEvent::Class())
                         ProcessTrueMesonCandidatesAOD(omegacand,gamma0,gamma1,gamma2);
                     }
+                    if(fDoMesonQA>0 && (omegacand->M()>0.4) && (omegacand->M()<1.2) ){
+                      TVector3 Boost = omegacand->BoostVector();
```

Comment:
The following lines from here seem to appear at different locations being (almost) identical - could you please put them into a function in order to avoid all the copy&paste?

Reply:
There should be two more commits where I changed that into a function and also adapted the code to the recently changed AliAODConversionPhoton::SetIsCaloPhoton().

Reply:
Something went wrong apparently, you need to fix the PR first

Reply:
Will try with a new PR. Sorry for the inconvenience.